src/components/ha-sidebar.ts

```diff
@@ -475,15 +475,17 @@ class HaSidebar extends LitElement {
 
       .menu {
         box-sizing: border-box;
-        height: 65px;
+        height: 64px;
         display: flex;
         padding: 0 12px;
-        border-bottom: 1px solid transparent;
         white-space: nowrap;
         font-weight: 400;
         color: var(--primary-text-color);
-        border-bottom: 1px solid var(--divider-color);
-        background-color: var(--primary-background-color);
+        border-bottom: 1px solid var(--divider-color, transparent);
+        background-color: var(
+          --sidebar-header-background-color,
+          var(--primary-background-color)
```

Comment:
Since `sidebar-header-background-color` is in derived styles it will always be defined and doesn't need the fallback to `primary-background-color`